model/project_parser.go

```diff
@@ -558,6 +559,7 @@ func GetProjectFromBSON(data []byte) (*Project, error) {
 // If reading from a version config, LoadProjectForVersion should be used to persist the resulting parser project.
 // opts is used to look up files on github if the main parser project has an Include.
 func LoadProjectInto(ctx context.Context, data []byte, opts GetProjectOpts, identifier string, project *Project) (*ParserProject, error) {
+	catcher := grip.NewBasicCatcher()
```

Comment:
Are you using a catcher so that you can still attempt to hit Translate? I think translate will fail if any of the above things went wrong, so I'm not sure it's worth doing that. It seems like enough to just return the parser project at whatever state it's at, with the error.